PluginScripts/plugin-heal-selection.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python3
```

Comment:
```suggestion
#!/usr/bin/env python3
```

Python doesn't have to be there but can still be in $PATH

Reply:
Oops!

Reply:
But wIth `env` it may find the wrong Python, e.g. in venvs where there are no `gi` package.

Reply:
Sometimes I just forgot I'm in a venv, and installed applications shouldn't break in a venv. People don't need to know if an application uses Python to use an application.

It is common to use `env python3` but it doesn't mean it is standard.

[PEP 394](https://peps.python.org/pep-0394/) says:

> For scripts that are only expected to be run in an activated virtual environment, shebang lines can be written as #!/usr/bin/env python, as this instructs the script to respect the active virtual environment.

This script uses `gi` which is uncommon to use in a venv, so it's more likely a system Python home should be used. That's `/usr/bin/python` on Linux. The shebang can be updated in the installation process, and various Python packaging systems do this.

Reply:
In NixOS where we don't have binaries in /usr/bin/ other than env, we patch the shebangs with the correct python env anyways so both versions work for us.

@WarpspeedSCP you can revert my suggestion.

Reply:
done.